### PR TITLE
(PCP-405) Make broker-ws-uris a config file only option

### PR DESCRIPTION
**Pull request by MikaelSmith**
Changes broker-ws-uris to only be settable via a config file. Also
clarifies how it and broker-ws-uri interract, and adds documentation in
the README.

### CODE REVIEW

**Comment by MikaelSmith** [comment]
The conclusion on how to handle multiple values on the command-line was either go comma-separated, or avoid it altogether. I went with avoid it.

An alternative would be to use comma-separated, and not introduce a new setting. Rather, `broker-ws-uri` could take either a string, or an array of strings. I'm not sure that's a great idea, as I think JSON schema would not support specifying multiple possible types for a property. However, the implementation would be quite a bit simpler, and shouldn't be ambiguous in our usage.

P.S. In general Websocket URIs could include commas in a query string, but I don't foresee us having a query string as part of broker URIs.

File: lib/src/configuration.cc [hunk 1]
```diff
@@ -618,13 +606,22 @@ void Configuration::parseConfigFile()
...
     for (const auto& key : config_json.keys()) {
         const auto& opt_idx = defaults_.get<Option::ByName>().find(key);
 
-        if (opt_idx == defaults_.get<Option::ByName>().end())
-            throw Configuration::Error {
-                lth_loc::format("field '{1}' is not a valid configuration variable",
-                                key) };
+        if (opt_idx == defaults_.get<Option::ByName>().end()) {
+            if (key == "broker-ws-uris") {
+                check_key_type(key, "Array", lth_jc::DataType::Array);
```
Comment (parisiale) [inline lib/src/configuration.cc]: Perhaps we should add a comment about `broker-ws-uris` being a configfile-only option.

**Comment by parisiale** [comment]
👍 
Just a comment request, as Configuration is pretty complex.

**Comment by MikaelSmith** [comment]
Added the comment.

**Pull request merged**